phpseclib/Net/SSH2.php

```diff
@@ -1411,7 +1417,7 @@ private function connect()
         if (!$this->send_kex_first) {
             $response = $this->get_binary_packet();
 
-            if (!strlen($response) || ord($response[0]) != NET_SSH2_MSG_KEXINIT) {
+            if (\is_bool($response) || !strlen($response) || ord($response[0]) != NET_SSH2_MSG_KEXINIT) {
```

Comment:
I'd say let's remove the `\`. I don't do them for any other function.

I did try to play around with this general idea a while back to see if it'd yield any speedups and it didn't:

https://github.com/terrafrost/phpseclib/commit/437e7f2cf2eb75ce42c3a33a2b29c57098946194

It's possible my testing was inadequate tho but idk lol

Reply:
It is definitely a speed up, but very (very) slight. I just have my IDE set to automatically add the slash. But following the coding style is more important.

Anyways, if this is worth commenting about, maybe we should implement a checker so it is not up for debate. php-cs-fixer has a rule for this: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/doc/rules/function_notation/native_function_invocation.rst

Reply:
I think that's a good idea! Submit a PR? 😁